### ISSUE TITLE
[Request] Option "Use torrent file" from the context menu in library

### ISSUE THREAD

**Issue by peter099** [elgatito/plugin.video.elementum#466]
What will be the purpose of this option?
When you add tv show or movie to library you go to the library, select the tv show episode or movie, and instead hitting enter to start elementum burst, open context menu, select "Use torrent file" option and browse for the torrent file.

What will be the benefits?
Elementum burst not always find results even if they exist on the trackers. This way you can search the trackers manually, download the torrent files, and use them from elementum. 
The torrent file will be cached so you have to browse for the torrent file only once. The next time you just hit enter and elementum will ask if you want to use the previous torrent.
It will be better option than "Add Torrent". With using the option "Add Torrent" you have to browse for the torrent file every time and if you are using season packs you have to go through list of episodes and select the exact episode, which is less convenient.

**Comment by elgatito** [comment]
Duplicate of #100
